lumicks/pylake/correlated_stack.py

```diff
@@ -241,54 +284,46 @@ def get_image(self, channel="rgb"):
 
         return np.stack([frame.data[slc] for frame in self], axis=0).squeeze()
 
+    @_deprecate_plot_arguments
     def plot(
         self,
-        frame=0,
         channel="rgb",
-        show_title=True,
         axes=None,
-        adjustment=ColorAdjustment.nothing(),
         *,
+        frame=0,
         image_handle=None,
+        adjustment=ColorAdjustment.nothing(),
+        show_title=True,
         **kwargs,
     ):
-        """Plot image from image stack
+        """Plot a frame from the image stack for the requested color channel(s)
 
         Parameters
         ----------
+        channel : {"red", "green", "blue", "rgb"}, optional
+            Color channel to plot.
```

Comment:
👍 nice that we're updating to the `{option1, option2, ...}` format as we're finding them

Reply:
😃